messaging/src/main/java/org/axonframework/commandhandling/callbacks/FailureLoggingCallback.java

```diff
@@ -30,30 +29,15 @@
  * @param <R> The return value of the command handler
  */
 public class FailureLoggingCallback<C, R> implements CommandCallback<C, R> {
-
-    private final CommandCallback<C, R> delegate;
     private final Logger logger;
 
     /**
-     * Initialize the callback to delegate calls to the given {@code delegate}, logging failures on a logger
-     * for this class (on warn level).
-     *
-     * @param delegate The command callback to forward invocations to
-     */
-    public FailureLoggingCallback(CommandCallback<C, R> delegate) {
```

Comment:
Although I think the chances are slim that users directly invoke this constructor, it is still a breaking change.
As such, we can't make this adjustment.

Reply:
Re-added the constructor